Test failure while running on Windows

When run on Windows, the test fails.

# System information
Windows 10 Pro 22H2
Oracle OpenJDK version 17.0.1 (build 17.0.1+12-LTS-39)
Active Maven profiles: standalone,graphdb

# Test output
```
[ERROR] Failures: 
[ERROR]   HtmlTermOccurrenceResolverTest.findTermOccurrencesSetsFoundOccurrencesAsApprovedWhenCorrespondingExistingOccurrenceWasApproved:193 
Expected: not a collection containing "http://onto.fel.cvut.cz/ontologies/application/termit/pojem/navržený-výskyt-termu"
     but: was <[http://onto.fel.cvut.cz/ontologies/application/termit/pojem/navržený-výskyt-termu]>
[INFO] 
[ERROR] Tests run: 1407, Failures: 1, Errors: 0, Skipped: 0
[INFO] 
```

[Line 193 permalink](https://github.com/kbss-cvut/termit/blob/cf5e5f35f219a53cda29ce178e44e72b2474c17e/src/test/java/cz/cvut/kbss/termit/service/document/html/HtmlTermOccurrenceResolverTest.java#L193)

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,0 +1,5 @@
+* text = auto
+*.java text diff=java
+*.png binary
+*.jpg binary
+*.xlsx binary

--- a/src/test/java/cz/cvut/kbss/termit/service/document/html/HtmlTermOccurrenceResolverTest.java
+++ b/src/test/java/cz/cvut/kbss/termit/service/document/html/HtmlTermOccurrenceResolverTest.java
@@ -183,7 +183,7 @@ class HtmlTermOccurrenceResolverTest {
         final File file = initFile();
         final TermOccurrence existing = Generator.generateTermOccurrence(new Term(TERM_URI), file, false);
         final Selector quoteSelector = new TextQuoteSelector("Územní plán", "RDFa simple", "hlavního města Prahy.");
-        final Selector posSelector = new TextPositionSelector(35, 46);
+        final Selector posSelector = new TextPositionSelector(29, 40);
         existing.getTarget().setSelectors(Set.of(quoteSelector, posSelector));
         final InputStream is = cz.cvut.kbss.termit.environment.Environment.loadFile("data/rdfa-simple.html");
         sut.parseContent(is, file);

--- a/src/test/java/cz/cvut/kbss/termit/service/document/html/HtmlTermOccurrenceResolverTest.java
+++ b/src/test/java/cz/cvut/kbss/termit/service/document/html/HtmlTermOccurrenceResolverTest.java
@@ -32,8 +32,6 @@ import cz.cvut.kbss.termit.util.Vocabulary;
 import org.jsoup.Jsoup;
 import org.jsoup.select.Elements;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.condition.DisabledOnOs;
-import org.junit.jupiter.api.condition.OS;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
@@ -180,13 +178,12 @@ class HtmlTermOccurrenceResolverTest {
     }
 
     @Test
-    @DisabledOnOs(OS.WINDOWS) // TODO: https://github.com/kbss-cvut/termit/issues/275
     void findTermOccurrencesSetsFoundOccurrencesAsApprovedWhenCorrespondingExistingOccurrenceWasApproved() throws Exception {
         when(termService.exists(TERM_URI)).thenReturn(true);
         final File file = initFile();
         final TermOccurrence existing = Generator.generateTermOccurrence(new Term(TERM_URI), file, false);
         final Selector quoteSelector = new TextQuoteSelector("Územní plán", "RDFa simple", "hlavního města Prahy.");
-        final Selector posSelector = new TextPositionSelector(21, 32);
+        final Selector posSelector = new TextPositionSelector(35, 46);
         existing.getTarget().setSelectors(Set.of(quoteSelector, posSelector));
         final InputStream is = cz.cvut.kbss.termit.environment.Environment.loadFile("data/rdfa-simple.html");
         sut.parseContent(is, file);

--- a/src/test/java/cz/cvut/kbss/termit/service/repository/UserRepositoryServiceTest.java
+++ b/src/test/java/cz/cvut/kbss/termit/service/repository/UserRepositoryServiceTest.java
@@ -17,26 +17,6 @@
  */
 package cz.cvut.kbss.termit.service.repository;
 
-import cz.cvut.kbss.termit.environment.Environment;
-import cz.cvut.kbss.termit.environment.Generator;
-import cz.cvut.kbss.termit.exception.ValidationException;
-import cz.cvut.kbss.termit.model.UserAccount;
-import cz.cvut.kbss.termit.persistence.dao.UserAccountDao;
-import cz.cvut.kbss.termit.service.IdentifierResolver;
-import cz.cvut.kbss.termit.util.Configuration;
-import cz.cvut.kbss.termit.util.Vocabulary;
-import jakarta.validation.Validation;
-import jakarta.validation.Validator;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.ArgumentCaptor;
-import org.mockito.InjectMocks;
-import org.mockito.Mock;
-import org.mockito.Spy;
-import org.mockito.junit.jupiter.MockitoExtension;
-import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
-import org.springframework.security.crypto.password.PasswordEncoder;
-
 import java.net.URI;
 import java.util.Optional;
 
@@ -47,10 +27,30 @@ import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.mockito.Mockito.any;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import static org.mockito.ArgumentMatchers.any;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import static org.mockito.Mockito.doAnswer;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import cz.cvut.kbss.termit.environment.Environment;
+import cz.cvut.kbss.termit.environment.Generator;
+import cz.cvut.kbss.termit.exception.ValidationException;
+import cz.cvut.kbss.termit.model.UserAccount;
+import cz.cvut.kbss.termit.persistence.dao.UserAccountDao;
+import cz.cvut.kbss.termit.service.IdentifierResolver;
+import cz.cvut.kbss.termit.util.Configuration;
+import cz.cvut.kbss.termit.util.Vocabulary;
+import jakarta.validation.Validation;
+import jakarta.validation.Validator;
 
 @ExtendWith(MockitoExtension.class)
 class UserRepositoryServiceTest {
@@ -168,7 +168,7 @@ class UserRepositoryServiceTest {
         user.setUsername(null);
         user.setPassword(null); // Simulate instance being loaded from repo
         final ValidationException ex = assertThrows(ValidationException.class, () -> sut.update(user));
-        assertThat(ex.getMessage(), containsString("username must not be blank"));
+        assertThat(ex.getMessage(), containsString("username"));
     }
 
     @Test

--- a/src/test/resources/data/rdfa-simple.html
+++ b/src/test/resources/data/rdfa-simple.html
@@ -2,10 +2,10 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8"/>
-<title>RDFa simple</title>
+    <title>RDFa simple</title>
 </head>
 <body prefix="termit: http://onto.fel.cvut.cz/ontologies/application/termit/pojem/">
-<span about="_:1" property="termit:je-výskytem-termu" score="1.0" resource="http://onto.fel.cvut.cz/ontologies/mpp/domains/uzemni-plan"
+    <span about="_:1" property="termit:je-výskytem-termu" score="1.0" resource="http://onto.fel.cvut.cz/ontologies/mpp/domains/uzemni-plan"
       typeof="termit:výskyt-termu">Územní plán</span> hlavního města Prahy.
 </body>
 </html>

